3.analyze_data/nbconverted/make_volcano_predicted_probabilities.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Compare phenotypic expression among treatments with volcano plot visualizations
+# We compare phenotypic expression for each (phenotype, statistical test, model type, treatment type) combination
+
+# In[ ]:
+
+
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+
+# ## Find the root of the git repo on the host system
+
+# In[ ]:
+
+
+# Get the current working directory
+cwd = pathlib.Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
+
+
+# ## Define the input dataframe from the parquet path
+
+# In[ ]:
+
+
+compdf = pd.read_parquet("statistical_test_comparisons/comparison_results.parquet")
+
+
+# ## Define and create output path
+
+# In[ ]:
+
+
+output_path = pathlib.Path("figures")
+
+output_path.mkdir(parents=True, exist_ok=True)
+
+
+# In[1]:
+
+
+# Bernoulli adjustment
+alpha = 0.05 / len(compdf)
+
+# Remove the zero values after the p-value adjustment
+compdf = compdf.loc[compdf["p_value"] > 0]
+significance_threshold = -np.log10(alpha)
+
+# Define the unique groups to plot
+unique_comparisons = set(zip(compdf["phenotype"], compdf["treatment_type"], compdf["model_type"], compdf["statistical_test"]))
+
+# Specify colors according to categories
+colors = {
```

Comment:
I think it might be good to include a legend for these plots as to what these colors mean. To someone just looking at this plot without context is a bit hard to follow and the title is pretty long also.

Maybe it would help to have a legend that shows Red = Significant and differential from DMSO.